src/components/Button/Button.css

```diff
@@ -1,108 +1,119 @@
 .uir-button {
-    position:relative;
-    border:0; /* override browser's default */
-    color:rgba(47,40,51,0.6);
-    min-width:50px;
+    position: relative;
+    border: 0; /* override browser's default */
+    color: rgba(47,40,51,.6);
```

Comment:
There's an immediately obvious case for using SASS (or another CSS preprocessor) here, in that we're repeating the colour many, many times (albeit with different opacity).

In SASS this could be:

```sass
$buttonGrey: rgba(47, 40, 51);

$buttonFgDefault: transparentize($buttonGrey, 0.6);
$buttonBgDefault: transparentize($buttonGrey, 0.05);

$buttonFgHover: $buttonGrey;
$buttonBgHover: transparentize($buttonGrey, 0.1);
```